docs/developer/extending/apps/building-payment-app.mdx

```diff
@@ -4,158 +4,73 @@ title: How to Build a Payment App
-| 1. Initiate the payment process in the drop-in | ---                                                                                                                                     | ---                                                                                                                |
-| 2. Confirm the payment via 3D Secure           | [`CHARGE_ACTION_REQUIRED`](api-reference/payments/enums/transaction-event-type-enum.mdx#transactioneventtypeenumcharge_action_required) | The payment requires additional action to be completed. **The drop-in token is needed to initialize the payment.** |
-| 3. Charge the payment                          | ---                                                                                                                                     | ---                                                                                                                |
-| 3a. Charge successful                          | [`CHARGE_SUCCESS`](api-reference/payments/enums/transaction-event-type-enum.mdx#transactioneventtypeenumcharge_success)                 | The payment was successful.                                                                                        |
-| 3b. Charge failed                              | [`CHARGE_FAILURE`](api-reference/payments/enums/transaction-event-type-enum.mdx#transactioneventtypeenumcharge_failure)                 | The payment failed, for example, due to insufficient funds.                                                        |
-| 4. Redirect the user to the result page        | ---                                                                                                                                     | ---                                                                                                                |
-
-### Webhooks
-
-#### Transaction Initialize Session
-
-With our transaction events mapped out, let's determine which webhooks we need to implement to handle these events.
-
-When exploring the [Transaction Events](developer/extending/webhooks/synchronous-events/transaction.mdx) documentation, you may notice that three webhooks can result in the `CHARGE_SUCCESS` event: [`TRANSACTION_CHARGE_REQUESTED`](developer/extending/webhooks/synchronous-events/transaction.mdx#transaction-charge), [`TRANSACTION_INITIALIZE_SESSION`](developer/extending/webhooks/synchronous-events/transaction.mdx#initialize-transaction-session) and [`TRANSACTION_PROCESS_SESSION`](developer/extending/webhooks/synchronous-events/transaction.mdx#process-transaction-session).
-
-There are two reasons for why **we are not going** with the `TRANSACTION_CHARGE_REQUESTED` webhook:
-
-1. It is designed for the staff users to charge the payment manually. Since we want a checkout user (possibly unauthenticated) to be able to pay for the order, it is not a good fit.
-2. It does not allow the return of the `CHARGE_ACTION_REQUIRED` event status.
+### Prerequisites
```

Comment:
**suggestion (non-blocking)**: Right now, we have two "prerequisites" sections. The latter one also contains "knowledge" prerequisites:

> - Basic understanding of [Saleor Apps](developer/extending/apps/overview.mdx)
> - Familiarity with [Next.js](https://nextjs.org/)

I suggest either merging those two into this "knowledge prerequisites" section or look for a different name for those sections.